FreeRTOS/FreeRTOS
# Kernel tasks.c Unit Test

Kernel tasks.c Unit Test

Description
-----------
Kernel tasks.c Unit Test
coverage 96%

Test Steps
-----------
command:
```
$ make run
....
================== SUMMARY =====================

Tests Passed  : 233
Tests Failed  : 0
Tests Ignored : 0
```
Coverage results:
```
$ make coverage
..
Overall coverage rate:
  lines......: 96.1% (871 of 906 lines)
  functions..: 96.1% (73 of 76 functions)
  branches...: 95.6% (417 of 436 branches)
```


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
